### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by michaelizergit** [newyjp/JPVideoPlayer#240]
Greetings, @newyjp 

I am a designer in development and an open source enthusiast who would love to contribute to your project by designing a logo for it. I found your project exploring GitHub and found it interesting. About the process, we would be working together to create a logo that fits best with your project. The process is, of course, totally free and you can quit whenever you want. In case you agree, it would be useful if you had any ideas (shapes, colors, etc) to have something to start with.

Best regards!

**Comment by newyjp** [comment]
Hi there, you can try it following yourself ideas and, send your design result to me, thanks!

**Comment by michaelizergit** [comment]
Of course, I'll start preparing some sketches right away and will come back as soon as I finish them.
